_rules/2045c3.md

```diff
@@ -0,0 +1,103 @@
+---
+id: 2045c3
+name: alert Role or Live Region Identify Input Error
+rule_type: atomic
+
+description: |
+  This rule checks that a container with `role=alert` or a live region are used to identify input errors
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag-technique:ARIA19: # Using ARIA role=alert or Live Regions to Identify Errors
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+
+authors:
+  - Carlos Duarte
+  - João Vicente
+---
+
+## Applicability
+
+This rule applies to any [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `html` element that has at least one [input element](https://www.w3.org/TR/html52/sec-forms.html#the-input-element).
```

Comment:
I would suggest aligning the applicability (of all these rules) with the two other rules about form fields: 

- Form field has accessible name, https://act-rules.github.io/rules/e086e5
- Form field label is descriptive, https://act-rules.github.io/rules/cc0f0a

That would mean replacing: 

> ... that has at least one [input element](https://www.w3.org/TR/html52/sec-forms.html#the-input-element).

...with something like: 

> ... that has at least one element that has one of the following [semantic roles](#semantic-role): `checkbox`, `combobox` (`select` elements), `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider, spinbutton`, `switch`, `textbox`.
> 
> **Note**: The list of roles is derived by taking all the [ARIA](https://www.w3.org/TR/wai-aria-1.1/) 1.1 roles that:
> 
> - have a [semantic roles](#semantic-role) that inherits from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and
> - does not have a [required context](https://www.w3.org/TR/wai-aria/#scope) role that itself inherits from one of those roles.
> - The `option` role is not part of the list of applicable roles, because it does not meet the definition of a [User interface component](https://www.w3.org/TR/WCAG21/#dfn-user-interface-components). This means [WCAG 2.1](https://www.w3.org/TR/WCAG21/) does not require it to have an accessible name.

Another question is whether it should only apply to inputs that are [included in the accessibility tree](#included-in-the-accessibility-tree), or if it should also include those that are only [visible](#visible).